packages/@lwc/engine-core/src/framework/base-lightning-element.ts

```diff
@@ -215,7 +214,7 @@ export const LightningElement: LightningElementConstructor = function (
     }
 
     // Making the component instance a live value when using Locker to support expandos.
-    defineProperty(component, lockerLivePropertyKey, EmptyObject);
+    (this as any)[lockerLivePropertyKey] = undefined;
```

Comment:
With a standard property assignment, the symbol is now writable and configurable. I would defer this to @caridy.

As a side note, we have a utility called [`fastDefineProperty`](https://github.com/salesforce/lwc/blob/master/packages/@lwc/synthetic-shadow/src/shared/node-ownership.ts#L19-L33) in synthetic shadow which preserves the best of both worlds: restrictive in dev and fast in production.

Reply:
I saw `fastDefineProperty` but it would need to move to `@lwc/shared`. I'm happy with either direction as long as @caridy approves.

Reply:
ok... I believe this is safe... a test should be added in Locker (cc @rwaldron) to validate that a class doing the assignment above coming from a sandbox can be extended, and in the extension, the value is deleted from `this`, and validate that in fact the instance is live. I believe that will work just fine, but better to test the behavior.

The reason why I think it will work is because when the subclass attempts to touch the `this`, the proxy is created, and the delete trap will make determiniiation about the nature of the proxy, live or static, and then carry on the operation, which means deleting that property will have no implications at all on the subclass, basically, it is too late to delete it.